app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,54 @@
+class RecipesController < ApplicationController
+  def index
+    @recipes = Recipe.all
+  end
+
+  def show
+    @recipe = Recipe.find(params[:id])
+  end
+
+  def new
+    redirect_to recipes_path, flash: { alert: 'Please sign up or login!' } unless current_user
+
+    @recipe = Recipe.new
+  end
+
+  def create
+    @new_recipe = current_user.recipes.new(recipe_params)
+    if @new_recipe.save!
+      redirect_to recipes_path, flash: { alert: 'Your recipe is saved' }
+    else
+      redirect_to new_recipe_path, flash: { alert: 'Could not save your recipe' }
+    end
+  end
+
+  def destroy
+    @recipe = Recipe.find(params[:id])
+    @recipe.destroy!
+    flash[:notice] = 'You have deleted the food!'
+    redirect_to recipes_path
+  end
```

Comment:
- [ ]  @team great work adding a destroy method to delete a recipe, however, it would be better to provide both success and fail paths, to cover situations when there is a bug and the food is not recipe